cpp/src/Slice/Parser.h

```diff
@@ -243,7 +243,7 @@ class Comment : public ::IceUtil::SimpleShared
     StringList seeAlso() const;   // Targets of @see tags.
 
     StringList returns() const;                           // Description of an operation's return value.
-    std::map<std::string, StringList> parameters() const; // Parameter descriptions for an op. Key is parameter name.
+    std::map<std::string, StringList> params() const; // Parameter descriptions for an op. Key is parameter name.
```

Comment:
I think it might be worth leaving this method named 'parameters'.
`Comment::parameters()` and `Operation::parameters()` are completely different functions, and having them be different names would make it much easier to distinguish them at a glance.

Reply:
The goal is consistency, and use params everywehere.